lib/components/user/notification-prefs-pane.tsx

```diff
@@ -22,91 +21,109 @@ interface Props extends FormikProps<User> {
+  }
+  label::first-letter {
+    text-transform: uppercase;
+  }
+  label + span {
+    color: ${GRAY_ON_WHITE};
+  }
 `
 
 /**
  * User notification preferences pane.
  */
 const NotificationPrefsPane = ({
-  loggedInUser,
   onRequestPhoneVerificationCode,
   onSendPhoneVerificationCode,
   phoneFormatOptions,
   values: userData // Formik prop
 }: Props): JSX.Element => {
-  const { email, isPhoneNumberVerified, phoneNumber } = loggedInUser
-  const { notificationChannel } = userData
+  const { email, isPhoneNumberVerified, phoneNumber, pushDevices } = userData
 
   return (
-    <div>
-      <p>
-        <FormattedMessage id="components.NotificationPrefsPane.description" />
-      </p>
-      <FormGroup>
-        <ButtonGroup>
-          <legend>
-            <FormattedMessage id="components.NotificationPrefsPane.notificationChannelPrompt" />
-          </legend>
-          {allowedNotificationChannels.map((type) => {
-            // TODO: If removing the Save/Cancel buttons on the account screen,
-            // persist changes immediately when onChange is triggered.
-            const inputId = `notification-channel-${type}`
-            const isChecked = notificationChannel === type
-            return (
-              <Fragment key={type}>
-                {/* Note: labels are placed after inputs so that the CSS focus selector can be easily applied. */}
-                <Field
-                  id={inputId}
-                  name="notificationChannel"
-                  type="radio"
-                  value={type}
-                />
-                <label
-                  className={
-                    isChecked ? 'btn btn-primary active' : 'btn btn-default'
-                  }
-                  htmlFor={inputId}
-                >
-                  {type === 'email' ? (
+    <FieldSet>
+      <legend>
+        <FormattedMessage id="components.NotificationPrefsPane.notificationChannelPrompt" />
+      </legend>
+      {allowedNotificationChannels.map((type) => {
+        // TODO: If removing the Save/Cancel buttons on the account screen,
+        // persist changes immediately when onChange is triggered.
+        const inputId = `notification-channel-${type}`
+        const inputDescriptionId = `${inputId}-description`
+        return (
+          <NotificationOption key={type}>
+            <span>
+              <Field
+                aria-describedby={inputDescriptionId}
+                // TODO: Check this condition.
+                disabled={type === 'push' && !pushDevices}
+                id={inputId}
+                name="notificationChannel"
+                type="checkbox"
+                value={type}
+              />
+            </span>
+            <span>
+              {type === 'email' ? (
+                <>
+                  <label htmlFor={inputId}>
                     <FormattedMessage id="common.notifications.email" />
-                  ) : type === 'sms' ? (
+                  </label>
```

Comment:
Might be a bit cleaner to move the `label` out of these blocks, and conditionally render the i18n id

Reply:
Good idea! Updated in https://github.com/opentripplanner/otp-react-redux/pull/899/commits/9ae5bfda7a0b9fff321465c130bb0810da4a105c. It also unveiled an oversight in `TripNotificationsPane` that I fixe with https://github.com/opentripplanner/otp-react-redux/pull/899/commits/bf54d586f1847e3bbe1b012118096ef98f0edb00.